src/mesh/http/WiFiAPClient.cpp

```diff
@@ -254,35 +252,24 @@ bool initWifi(bool forceSoftAP)
                 WiFi.setHostname(ourHost);
                 WiFi.onEvent(WiFiEvent);
 
-#if !CONFIG_IDF_TARGET_ESP32S3
                 // This is needed to improve performance.
-                esp_wifi_set_ps(WIFI_PS_NONE); // Disable radio power saving
-#endif
+                // esp_wifi_set_ps(WIFI_PS_NONE); // Disable radio power saving
 
                 WiFi.onEvent(
                     [](WiFiEvent_t event, WiFiEventInfo_t info) {
                         Serial.print("\nWiFi lost connection. Reason: ");
-                        #if CONFIG_IDF_TARGET_ESP32S3
                         Serial.println(info.wifi_sta_disconnected.reason);
-                        wifiDisconnectReason = info.wifi_sta_disconnected.reason;
-                        #else
-                        Serial.println(info.disconnected.reason);
+
                         /*
                            If we are disconnected from the AP for some reason,
                            save the error code.
 
                            For a reference to the codes:
                              https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/wifi.html#wi-fi-reason-code
                         */
-                        wifiDisconnectReason = info.disconnected.reason;
-                        #endif
-
+                        wifiDisconnectReason = info.wifi_sta_disconnected.reason;
                     },
-#if CONFIG_IDF_TARGET_ESP32S3
                     WiFiEvent_t::ARDUINO_EVENT_WIFI_STA_DISCONNECTED);
-#else
-                    WiFiEvent_t::SYSTEM_EVENT_STA_DISCONNECTED);
```

Comment:
should this not be `WiFiEvent_t::ARDUINO_EVENT_WIFI_STA_DISCONNECTED);` ?

Reply:
it is - the else branch is the old API and is removed by this PR :-)